addons/io_hubs_addon/preferences.py

```diff
@@ -165,6 +165,101 @@ def execute(self, context):
         return {'FINISHED'}
 
 
+def reload_user_components():
+    from .components.components_registry import unload_user_components, load_user_components
+    unload_user_components()
+    load_user_components()
```

Comment:
This reload_user_components function doesn't seem to be used anywhere.  Is there supposed to be a refresh button in the preferences that calls this?  (Currently there doesn't seem to be any way to pick up the definitions from a new folder without saving the preferences and restarting Blender.)